lib/trade_tariff_frontend.rb

```diff
@@ -91,6 +91,10 @@ def js_sentry_dsn
     ENV.fetch('JS_SENTRY_DSN', '')
   end
 
+  def rules_of_origin_enabled?
+    %w[1 true yes].include? ENV['RULES_OF_ORIGIN_ENABLED'].to_s.downcase
```

Comment:
I reckon we can just pick one of these since we're in control of setting it and there are only two of us? I'd go for `true` and `false` and be consistent with things like RAILS_SERVE_STATIC_FILES, etc

Reply:
Totally non-blocking

Reply:
Cool, will change